compiler/rustc_target/src/spec/bpf_base.rs

```diff
@@ -19,6 +19,10 @@ pub fn opts(endian: Endian) -> TargetOptions {
         obj_is_bitcode: true,
         requires_lto: false,
         singlethread: true,
+        // When targeting the `v3` cpu in llvm, 32-bit atomics are also supported.
```

Comment:
Is there any reason why anybody would want to use an older `target-cpu` with BPF targets?

Reply:
Support older kernels